<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="choose_file_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose file</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="selected_file_text">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selected_file_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Currently selected file:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>181</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
